exporters/otlp/test/otlp_http_log_exporter_test.cc

```diff
@@ -256,6 +259,8 @@ TEST_F(OtlpHttpLogExporterTestPeer, ExportBinaryIntegrationTest)
         http_client::nosend::Response response;
         callback.OnResponse(response);
```

Comment:
don't we need `response.Finish(callback);` here?

Reply:
I want to reuse it to trigger `OnEvent` and call `OnResponse`. The codes do not trigger `OnEvent` before.